rating-service/src/test/java/com/lohika/jclub/rating/service/RatingServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package com.lohika.jclub.rating.service;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringRunner.class)
```

Comment:
Do we need to test the service? The previous test is testing MVC and this service.